hub/analytics.py

```diff
@@ -58,9 +51,24 @@ def imported_data_count_by_area(
             .order_by("-count")
         )
 
+    def imported_data_by_area(
+        self,
+        postcode_io_key: str = None,
+        layer_ids: List[str] = None,
```

Comment:
Kinda seems like either of these args should be default None since they're both required, no?

Reply:
Yeah, probably, but I'm going to leave it for now, just for the sake of timesaving!